cli/man/splinter-circuit-show.1.md

```diff
@@ -104,6 +105,42 @@ Proposal to create: 01234-ABCDE
                 AA01
 ```
 
+The information displayed below will appear the same on all proposed member
+nodes for a circuit proposal to disband an existing circuit.
+If all member nodes vote to accept disbanding the circuit, the
+`splinter-circuit-show` command will display the same information, without the
+`Vote` as all nodes would have accepted the proposal. If any of the member
+nodes vote to reject disbanding the circuit, the proposal will not be viewable
```

Comment:
"the proposal will not be viewable" this is kinda misleading because really the proposal will no longer exist.

Reply:
Updated here to say the `the proposal will be removed and is not viewable by any nodes`, and also clarified the wording above where the same sentence was used (and the same circumstances pertain, the proposal is not only no longer viewable, but has been removed)